src/tests/test_trivia.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Please move this file into a new `src/tests/service` directory (like [this](https://github.com/MoritzHayden/bosco/tree/main/src/tests/util))